source/fundamentals/logging.txt

```diff
@@ -10,216 +10,21 @@ Logging
    :depth: 1
    :class: singlecols
 
-Overview
---------
+.. note::
 
-This guide shows you how to configure the driver logger which is responsible
-for reporting events that occur while your application is running. You can
-set the logger level, specify logging settings for individual classes, and
-provide a custom logger implementation.
+  The driver doesn't use the logger in this version. Attempting to use
+  prior settings in this version won't print anything. Instead, see our
+  :doc:`command monitoring guide <monitoring/command-monitoring>`. 
 
-You should read this guide if you need to customize the default behavior of the
-driver logger or want to explore the information provided by the logger.
+Temporary Alternative
+---------------------
 
-Log Levels
-----------
+The Node.js team is currently working on rewriting the logger. In the
+meantime, you can log monitor events in the console using the following snippet:
```

Comment:
```suggestion
meantime, you can log monitor events using the following snippet:
```
Nit: I think "in the console" doesn't add much here, and I think it might be a little confusing as "console" I think has associations with consoles in web browsers more than `stdout`. This is super nitty at this point though and I think everyone gets what is being said.